core/src/main/java/org/kiwix/kiwixmobile/core/extensions/HeaderizableListExtensions.kt

```diff
@@ -0,0 +1,21 @@
+package org.kiwix.kiwixmobile.core.extensions
+
+typealias HeaderizableList<T> = List<T>
```

Comment:
Typealiases don't work for scoping extension functions, it is still on the original type

![image](https://user-images.githubusercontent.com/3358282/82916836-76ff1600-9f6a-11ea-8221-7555bc5b380e.png)

Inline classes do allow the scoping though